packages/twenty-server/src/engine/api/graphql/workspace-query-runner/listeners/telemetry.listener.ts

```diff
@@ -25,6 +25,7 @@ export class TelemetryListener {
             data: {
```

Comment:
remove type: track and pass `action` instead with `payload.name` as its value? eventName isn't part of your schema is it?